File: 1421-find-numbers-with-even-number-of-digits/1421-find-numbers-with-even-number-of-digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int count = 0;
        for(int num : nums){
        if(evenDigits(num)) count++;
        }
        return count;
    }
    public boolean evenDigits(int num) {
        int digiCount = 0;
        while(num > 0){
            digiCount++;
            num = num / 10;
        }
        return digiCount % 2 == 0;
    }
}

File: 1421-find-numbers-with-even-number-of-digits/1421-find-numbers-with-even-number-of-digits.c
int order(int n){
    int c = 0;
    while(n){n /= 10; c++;}
    return c; 
}
int findNumbers(int* arr, int n) {
    int c = 0;
    for(int i = 0; i < n; i++) if(order(arr[i]) % 2 == 0) c++;
    return c;
}